components/placeholder/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Dashicon from 'components/dashicon';
+
+function Placeholder( { icon, children, label, instructions, className, ...additionalProps } ) {
+	const classes = classnames( 'placeholder', className );
+
+	return (
+		<div { ...additionalProps } aria-label={ label } className={ classes }>
+			<div className="placeholder__label">
+				<Dashicon icon={ icon } />
+				{ label }
+			</div>
+			{ instructions
```

Comment:
You can avoid the ternary like this:

`{ !! instructions && <div className="placeholder__instructions">{ instructions }</div> }`